src/shared_modules/indexer_connector/src/monitoring.hpp

```diff
@@ -27,6 +28,7 @@
 #include <vector>
 
 constexpr auto INTERVAL = 60u;
+constexpr auto IC_MONITORING {"indexer-connector"};
```

Comment:
Nitpick: maybe if we move the `IC_NAME` constant, we can avoid declaring two expressions with the same value.

Reply:
I didn't like the idea of moving there, but I can do that, the NOT_USED flag should be fixed in http-request but I'd do the same if you agree.

Reply:
It was just a nitpick.
What you decide is Ok.